_schemaV2.graphql

```diff
@@ -7646,6 +7646,11 @@ type CurrentEvent {
 # Date in YYYY-MM-DD format
 scalar Date
 
+# A date-time string at UTC, such as 2007-12-03T10:15:30Z, compliant with the
+# `date-time` format outlined in section 5.6 of the RFC 3339 profile of the ISO
+# 8601 standard for representation of dates and times using the Gregorian calendar.
+scalar DateTime
```

Comment:
**question**: can we reuse [this scalar type](https://github.com/artsy/metaphysics/blob/057edc54a6e4c75a61b4ceefe93348bb257f3d37/_schemaV2.graphql#L10370)?

Reply:
I am not sure how to do it. If I understand it correctly - this type is coming directly from merged schema and not a part of `graphl` library.

Other options if you don't want to bring a new library:
- support our own GraphQLDateTime scalar, like [GraphQLLong](https://github.com/artsy/metaphysics/blob/main/src/lib/customTypes/GraphQLLong.ts)
- Use string instead 🤷 Gravity validates the type of `seen_at` param just in case

Also interesting to hear your thoughts @mzikherman

Reply:
Yea, that scalar type looks to be coming in from an upstream schema and is being merged in, so vanilla resolvers/mutations wouldn't have access to it (the code would need to live in the context of https://github.com/artsy/metaphysics/blob/37fc1983a57348dd5f33257c18875f6f179bf28f/src/lib/stitching/gravity/v2/stitching.ts).

In terms of bringing this dep. in for a new scalar - I would wonder what we've used for this in the past (we might have some example in Metaphysics of passing in a date field in a mutation?). I also might expect a timestamp field to support integers (in Ruby that would be converted back with `Time.at(...)` I think). That's what many Gravity endpoints that set timestamps do: https://github.com/artsy/gravity/blob/b82fac491b0b9847619ac079093f2fe326ae0fbc/app/api/v1/fairs_endpoint.rb#L29

Reply:
In Gravity, that would involve dropping the Grape stuff:  https://github.com/artsy/gravity/blob/b82fac491b0b9847619ac079093f2fe326ae0fbc/app/api/v1/me_notifications_endpoint.rb#L92, and just dealing with integers. Might be a bit easier/less ambiguous that way!

Reply:
thanks for feedback. Decided to keep it simple for now and use `string` instead